jobs/seed-github-ci.Jenkinsfile

```diff
@@ -80,6 +80,11 @@ node { repos.each { repo ->
                         }
                     }
                 }
+                it / sources / data / 'jenkins.branch.BranchSource' / buildStrategies {
+                    'jenkins.branch.buildstrategies.basic.ChangeRequestBuildStrategyImpl' {
+                        ignoreTargetOnlyChanges(true)
+                    }
+                }
```

Comment:
This looks good. One question:

If I open a PR and it sits for 6 weeks without any new force pushes how many times will CI get run on it?

Reply:
Assuming persistent storage, it should be just once (at the time you opened the PR or basically the first time Jenkins sees that PR).